nbstripout/_nbstripout.py

```diff
@@ -214,10 +214,12 @@ def _parse_size(num_str):
         raise ValueError(f"Unknown size identifier {num_str[-1]}")
 
 
-def install(git_config, install_location=INSTALL_LOCATION_LOCAL, python=None, attrfile=None):
+def install(git_config, install_location=INSTALL_LOCATION_LOCAL, python=None, attrfile=None, keep_args=None):
     """Install the git filter and set the git attributes."""
     try:
         filepath = f'"{PureWindowsPath(python or sys.executable).as_posix()}" -m nbstripout'
+        if keep_args:
+            filepath = filepath + ' '.join(keep_args)
```

Comment:
I think I'd rather configure these options via the git config directly i.e. if you want to use `keep_output` with the git filter, you set `git config --type bool filter.nbstripout.keepOutput true` and then we read that value to provide the default for `keep_output`.

Reply:
Makes sense. Here's my take on this:

- I am passing `extra_flags` to `install()` and `uninstall()` that keep track of the three flags `keep_output`, `keep_id` and `keep_count`, so they can be cleaned up.
- Since the underscores don't work for git config names, I am converting to camelCase
- If these flags are set in the cmd args, that will override whatever git config values are
- If the flags are not passed as the cmd args, do not exist in git config (e.g. running nbstripout without install), then they will be False